CHANGELOG.md

```diff
@@ -9,8 +9,13 @@ All notable changes to this project will be documented in this file.
 - Add `stackable_webhook` crate which provides utilities to create webhooks with TLS termination ([#730]).
 - Add `ConversionReview` re-export in `stackable_webhook` crate ([#749]).
 
+## Fixed
+
+- Fixed wrong schema (and thus CRD) for `config.affinity.nodeSelector`, making it unusable ([#752]).
```

Comment:
suggestion: The fix makes it _usable_ again. So I would suggest to just remove that note.

```suggestion
- Fix wrong schema (and thus CRD) for `config.affinity.nodeSelector` ([#752]).
```